images/linux/scripts/installers/2004/android.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+################################################################################
+##  File:  android.sh
+##  Desc:  Installs Android SDK
+################################################################################
+
+set -e
+
+# Source the helpers for use with the script
+source $HELPER_SCRIPTS/document.sh
+source $HELPER_SCRIPTS/os.sh
+
+# Set env variable for SDK Root (https://developer.android.com/studio/command-line/variables)
+ANDROID_ROOT=/usr/local/lib/android
+ANDROID_SDK_ROOT=${ANDROID_ROOT}/sdk
+echo "ANDROID_SDK_ROOT=${ANDROID_SDK_ROOT}" | tee -a /etc/environment
+
+# ANDROID_HOME is deprecated, but older versions of Gradle rely on it
+echo "ANDROID_HOME=${ANDROID_SDK_ROOT}" | tee -a /etc/environment
+
+# Create android sdk directory
+mkdir -p ${ANDROID_SDK_ROOT}
+
+# Download the latest command line tools so that we can accept all of the licenses.
+# See https://developer.android.com/studio/#command-tools
+wget -O android-sdk.zip https://dl.google.com/android/repository/sdk-tools-linux-4333796.zip
+unzip android-sdk.zip -d ${ANDROID_SDK_ROOT}
+rm -f android-sdk.zip
+
+# Add required permissions
+chmod -R a+X ${ANDROID_SDK_ROOT}
+
+if isUbuntu20 ; then
+    # Sdk manager doesn't work with Java > 8, set version 8 explicitly
+    sed -i "2i export JAVA_HOME=${JAVA_HOME_8_X64}" /usr/local/lib/android/sdk/tools/bin/sdkmanager
```

Comment:
Am I right that sdkmanager won't work for customers too and they will have to use the same workaround?

Reply:
It will work because we add the setting directly to the SDK manager executable script.